dependabot-auto-merge/global.d.ts

```diff
@@ -0,0 +1,3 @@
+declare global {
```

Comment:
I guess we have to commit `dist` since actions are loaded from the GitHub repo?

Reply:
That is right. We could also create another action that pushes dist to say, trunk-built but that'd be out of the scope of this PR. It's more of a nice to have though.